credsweeper/common/constants.py

```diff
@@ -140,3 +140,7 @@ class DiffRowType(Enum):
 
 # default value for config and ValuePemPatternCheck
 DEFAULT_PEM_PATTERN_LEN = 5
+
+# PEM x509 patterns
+PEM_BEGIN_PATTERN = "-----BEGIN"
+PEM_END_PATTERN = "-----END"
```

Comment:
Thank you for adding above constants but `PEM_END_PATTERN` does not used anywhere..
Could you please remove this?

```suggestion
# PEM x509 patterns
PEM_BEGIN_PATTERN = "-----BEGIN"
```